tools/data-handler/src/utils/clingo-parser.ts

```diff
@@ -0,0 +1,290 @@
+/**
+    Cyberismo
+    Copyright © Cyberismo Ltd and contributors 2024
+
+    This program is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General Public License version 3 as published by the Free Software Foundation.
+
+    This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public
+    License along with this program.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+interface Link {
```

Comment:
This interface should perhaps be somewhere elsewhere, as it is general link interface (even if there aren't any other users for it yet). Move it somewhere?

Reply:
Console logging the outputs should be a temporary solution anyway. Maybe it's a good reminder to fix it sooner or later? And as a debug log, I don't think it's a bad idea to specify it is clingo output, because then the debugger knows it is clingo and not clingraph for example.

Reply:
It actually differs from the general Link interface. I renamed it to CalculationLink. I think it should be in the same place as other interfaces related to calculations. Though we might want to move them too.